pkg/gui/presentation/icons/git_icons.go

```diff
@@ -59,3 +60,24 @@ func IconForRemote(remote *models.Remote) string {
 	}
 	return DEFAULT_REMOTE_ICON
 }
+
+
+func IconForChangeStatus(changeStatus string) string {
+	nfOctDiffAdded    := "\uf457 " // 
+	nfOctDiffModified := "\uf459 " // 
+	nfOctDiffRemoved  := "\uf458 " // 
+	nfOctDiffRenamed  := "\uf45a " // 
+
+	switch changeStatus {
```

Comment:
Conflicts are not handled.

Reply:
Good point. I think there's lots more areas I need to cover before we merge this PR.